camptix.js

```diff
@@ -230,6 +231,24 @@ var docCookies={getItem:function(e){return decodeURIComponent(document.cookie.re
 
 }(jQuery));
 
+/**
+ * Class for utility functions
+ */
+var Tix = new function() {
+
+	/**
+	 * Gets the currently selected payment option. If a new payment options
+	 * layout is implemented, then over write this function to select proper
+	 * payment option
+	 *
+	 * @returns {*|string}
+	 */
+	this.getSelectedPaymentOption = function() {
+		return jQuery('#tix [name="tix_payment_method"]').val() || 'stripe';
```

Comment:
I know this file has a lot of code that doesn't meet the WP JS coding standards (including some that I probably added). But since we're adding a new block here, let's try to get it right.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/

The main thing that comes to mind here is the need for extra spaces around arguments [#](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#arrays-and-function-calls)